debian8/guide/system/logging/package_rsyslog_installed.rule

```diff
@@ -18,4 +18,4 @@ references:
 
 ocil_clause: 'the package is not installed'
 
-ocil: '{{{ rpm_package_check(package="rsyslog") }}}'
+ocil: '{{{ package_check(package="rsyslog", pkg_system=pkg_system) }}}'
```

Comment:
could we please avoid passing the pkg_system? you should be able to get it from the environment

Reply:
+1

Reply:
I would love to. But as I have described in the PR description, I have not managed to get it working. I will give it a shot again, when I think of it, it should work without that argument.